js/Errors.ts

```diff
@@ -1,47 +1,25 @@
-/**
- * Expose an object with WebRTC Errors.
- */
-var Errors = (module.exports = {}),
-	/**
-	 * Local variables.
-	 */
-	IntermediateInheritor = function () {};
+function createErrorClass(name: string) {
+	return class NamedError extends Error {
+		name = name;
 
-IntermediateInheritor.prototype = Error.prototype;
-
-/**
- * Create error classes.
- */
-addError('InvalidStateError');
-addError('InvalidSessionDescriptionError');
-addError('InternalError');
```

Comment:
InternalError wasn't used at all

Reply:
yes, was for iosrtc native error. Im fine if we remove for now.